<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-A7UER8PU" failures="1" tests="6" name="Test" time="0.05" errors="0" timestamp="2022-02-08T18:14:42 IST">
  <testcase classname="org.test.com.Iretry_Test" name="userNameValidation" time="0.007">
    <skipped/>
  </testcase> <!-- userNameValidation -->
  <testcase classname="org.test.com.Iretry_Test" name="userNameValidation" time="0.001">
    <skipped/>
  </testcase> <!-- userNameValidation -->
  <testcase classname="org.test.com.Iretry_Test" name="userNameValidation" time="0.0">
    <skipped/>
  </testcase> <!-- userNameValidation -->
  <testcase classname="org.test.com.Iretry_Test" name="userNameValidation" time="0.001">
    <skipped/>
  </testcase> <!-- userNameValidation -->
  <testcase classname="org.test.com.Iretry_Test" name="userNameValidation" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [A] but found [a]">
      <![CDATA[java.lang.AssertionError: expected [A] but found [a]
at org.test.com.Iretry_Test.userNameValidation(Iretry_Test.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- userNameValidation -->
  <testcase classname="org.test.com.Iretry_Test" name="userPasswordValidation" time="0.001"/>
</testsuite> <!-- Test -->
